semantic-python/src/Language/Python/Core.hs

```diff
@@ -93,21 +122,30 @@ instance Compile (Py.AssertStatement Span)
 instance Compile (Py.Attribute Span)
 
 instance Compile (Py.Assignment Span) where
-  compile it@Py.Assignment { Py.left = Py.ExpressionList { Py.extraChildren = [lhs] }, Py.right = Just rhs } = do
-    target <- compile lhs
+  compileCC it@Py.Assignment
+    { Py.left = Py.ExpressionList
+      { Py.extraChildren =
+        [ Py.PrimaryExpressionExpression (Py.IdentifierPrimaryExpression (Py.Identifier { Py.bytes = name }))
```

Comment:
Rather than compiling the `ExpressionList` item directly, we pattern-match to get an `Identifer` out of it straightaway.

This simplifies the implementation, though we’ll have to return to the instance for `ExpressionList` at some later point.